src/scss/modules/buttons.scss

```diff
@@ -63,6 +63,11 @@
     border-color: color(light_blue);
     color: $white;
   }
+
+  .icon.fa-angle-down {
+    font-size: 12px;
+    vertical-align: -0.125em;
+  }
```

Comment:
Attempted to apply this to the `.button` class instead, but seemed less risky to just put it on `.button-primary` instead.

[.button-filter .icon.fa-angle-down](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/scss/modules/buttons.scss#L113) would be effected by that. And that class has some left and top margins that would make things complicated as well... as we'd only want that applied on the `.button-filter` button and not others.